clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/AbstractPartitionStateModel.java

```diff
@@ -144,7 +144,7 @@ public void rollbackOnError(Message message, NotificationContext context, StateT
        * When state transition fails, we shouldn't remove the corresponding database here since the database could
        * be either recovered by bounce/Helix Reset or completely dropped after going through 'ERROR' to 'DROPPED' state transition.
        */
-      stopConsumption();
+      stopConsumption(false);
```

Comment:
What happens to the CV when the replica is in `ERROR` state? Does CV represent `ERROR`, or does it continue carrying the previous state? If it's the latter, then should we also clear CV states here...? Either way, it seems a little tricky, and we might want to leave a comment to that effect?

Reply:
So once a replica reports `COMPLETED` in CV, it won't go back to `ERROR` state.
And if the replica becomes `ERROR` during bootstrapping, `ERROR` state will be reported in CV as well.

Reply:
Ok, so if the CV contains `ERROR`, that is fine, no need to remove it from the CV then.

Reply:
Shall we add this conclusion to the comment or do you prefer not to?

Reply:
Sure, will do.